homeassistant/components/plant/const.py

```diff
@@ -0,0 +1,40 @@
+"""Const for Plant."""
+from typing import Final
+
+DOMAIN: Final = "plant"
+
+PLANT_NAME = "plant_name"
+SENSORS = "sensors"
+
```

Comment:
These two constants are unused - you should remove them for now, and add them when you actually start making use of them.
```suggestion
```